gtsfm/two_view_estimator.py

```diff
@@ -115,10 +175,8 @@ def compute_relative_pose_metrics(
         Unit translation error, in degrees
     """
     R_error_deg = comp_utils.compute_relative_rotation_angle(i2Ri1_computed, i2Ti1_expected.rotation())
-
-    U_error_deg = comp_utils.compute_relative_unit_translation_angle(i2Ui1_computed, Unit3(i2Ti1_expected.translation()))
-
-    logger.debug("[Two View Estimator] Relative rotation error %f deg.", R_error_deg)
```

Comment:
i think these could be useful to keep printing these in some sort of debug mode. or at least i'd like to dump them to disk instead of remove them

Reply:
i prefer more verbosity now while we figure out all the issues

Reply:
We will dump them as JSON anyways. Will just keep them off stdout. What do you prefer?

Reply:
are they getting dumped to JSON now? if not, do you mind opening an issue for us to do that ASAP?

Reply:
I can make it part of this PR?

Reply:
sure, whichever you prefer